v2/app/src/main/java/com/google/firebase/example/makeitso/ui/settings/SettingsScreen.kt

```diff
@@ -81,23 +95,20 @@ fun SettingsScreen(
                 StandardButton(
                     label = R.string.sign_out,
                     onButtonClick = {
-                        viewModel.signOut(openHomeScreen)
+                        viewModel.signOut()
                     }
                 )
 
                 Spacer(Modifier.size(16.dp))
 
-                DeleteAccountButton(openHomeScreen, viewModel)
+                DeleteAccountButton(viewModel)
             }
         }
     }
 }
 
 @Composable
-fun DeleteAccountButton(
-    openHomeScreen: () -> Unit,
-    viewModel: SettingsViewModel = hiltViewModel()
-) {
+fun DeleteAccountButton(viewModel: SettingsViewModel = hiltViewModel()) {
```

Comment:
Not sure if it's a good idea to pass the viewModel to a button. Maybe we could pass a `onDeleteAccountClicked()` function instead?

Reply:
Done!